salt/metalk8s/salt/master/files/master_99-metalk8s.conf.j2

```diff
@@ -1,9 +1,9 @@
 file_roots:
-  metalk8s-{{ metal_version }}:
-    - /srv/scality/metalk8s-2.0/salt
+  {{ saltenv }}:
```

Comment:
Once we implement multiple versions and upgrades, this will no longer work correctly (but that's not  regression introduced by this PR, just a heads-up)

Reply:
Should I create a ticket tracking this as debt? I could split the master conf into multiple files, so that declaring file_roots is done in some `/etc/salt/master.d/99-metalk8s-{{ version }}.conf` which won't get overridden by other Salt environments.

Reply:
Created #800 to track this.